indico/modules/events/registration/fields/simple.py

```diff
@@ -405,13 +405,15 @@ def _picture_size_and_type(value):
             file = File.query.filter(File.uuid == value, ~File.claimed).first()
             if not file:
                 raise ValidationError('Invalid file')
+            if not file.meta.get('registration_picture_checked'):
+                raise ValidationError('Picture has not been properly validated')
             try:
                 with file.open() as f:
                     pic = Image.open(f)
             except OSError:
                 raise ValidationError(_('Invalid picture file.'))
-            if pic.format.lower() not in {'jpeg', 'png', 'gif', 'webp'}:
-                raise ValidationError(_('This field only accepts jpg, png, gif and webp picture formats.'))
+            if pic.format.lower() != 'jpeg':
+                raise ValidationError(_('This field only accepts jpg picture format.'))
```

Comment:
No need for i18n here, only people doing funny things will see this

```suggestion
                raise ValidationError('This field only accepts jpeg pictures.')
```

(this check is not strictl needed at all, since `registration_picture_checked` already implies it's a jpeg)

Reply:
changed